getting-started/bookshelf/pubsub_worker/worker.go

```diff
@@ -71,39 +72,37 @@ func main() {
 }
 
 func subscribe() {
```

Comment:
This name doesn't really reflect what it does.  Why not something like:

func process(sub *SubscriptionHandle)

?